### PR TITLE
feat: add support for new Billing on Arbitrum

### PR DESCRIPTION
**Pull request by pcarranzav**
This will remove references to the Billing contract(s) on Polygon, and replace them with the new ones on Arbitrum Goerli and Arbitrum One.

For now it's targeting the scratch testnet deployment from https://github.com/edgeandnode/billing-contracts/pull/27 - I will replace that with the proper testnet deployment when it's done, and also add the Arbitrum One deployment when we have one.

### CODE REVIEW

File: src/mappings/billing.ts
Comment (juanmardefago) [inline src/mappings/billing.ts]: I wouldn't delete entities unless it's completely required for stuff to work properly.
This could be easily swapped to a flag inside the entity that clearly outlines if an entity is `enabled` or not, which would make this explicit, but still retain entities in case we want to query old entities for their state

**Comment by juanmardefago** [comment]
I think, aside from that minor comment, everything looks good.
I would also consider waiting until L2 is merged and deployed for this code to be merged to main (basically, when the old code for billing is no longer relevant), since we can still deploy subgraphs for arbitrum without having to merge this to main, and keep the fact that it gets merged to main to signal that it's the code that it's gonna be used in production from that point onwards.

File: src/mappings/billing.ts
Comment (pcarranzav) [reply src/mappings/billing.ts]: I'll let @IainM22 weigh in on whether this affects the UI in any way, but otherwise this makes sense to me so I'd work on a fix

Comment (pcarranzav) [reply src/mappings/billing.ts]: Pushed this fix in the meantime, and I'll check with Iain to see if it works for him

**Review by juanmardefago — APPROVED** [review]

File: src/mappings/billing.ts
Comment (juanmardefago) [reply src/mappings/billing.ts]: Fix makes sense, so it should most likely work for @IainM22 :)

Comment (pcarranzav) [reply src/mappings/billing.ts]: Yup he confirmed so we're good to go 👍

**Review by tmigone — APPROVED** [review]

**Pull request merged**
